config/resources/mailing-lists.json

```diff
@@ -0,0 +1,62 @@
+{
+  "version": "0.1",
+  "prefixes": {
+    "nco": "http://www.semanticdesktop.org/ontologies/2007/03/22/nco#",
+    "vcard": "http://www.w3.org/2006/vcard/ns#"
+  },
+  "resources": {
+    "contact-list": {
```

Comment:
In lijn met de andere configuration files kan je hier best meervoud gebruiken. Dit gaat ook impact hebben op de `target` types die gebruikt worden.

Zoals het nu geconfigureerd is zou het API endpoint om de contactlijsten op te lijsten zijn: `/contact-list` terwijl `/contact-lists` meer in lijn is met REST. Je kan dit ook configureren met de `path` property (zie https://github.com/mu-semtech/mu-cl-resources#overview-of-the-keys) maar ik zou het dus op een analoge manier doen als in de andere files.